### PR TITLE
Adds a new combat medic hardsuit, exclusive to the blackmarket.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
I've actually really taken to enjoying using the blackmarket uplink recently, and I've had this idea in the backburner for awhile. Had to get some spriting out of my system to boot.

## About The Pull Request
_"I'm gonna knock you silly!"_
image

Adds a new hardsuit, the Combat Medic Hardsuit, which can only be obtained as a rare purchase from the Black Market Uplink. It has fairly weak armor values compared to either comitting to either a full on combat hardsuit (Security) or a full disease protection hardsuit (Medical). Ideally this would be used to combat against zombie or xeno outbreaks, due to the melee resistance in combination with the biological resistance. However, this comes at a fairly hefty cost, ranging from 5.5k-7k from the Black Market, and only a single one can be bought. Hopefully this isn't too powerful/weak, but we'll get to that bridge when we build it.

## Why It's Good For The Game

Adding more content to the black market, especially uncommon content, helps diversify the purchase pool. Also, adds a tiny new way to protect against game-ending threats like zombies and xenos without needing to rob xenobio/viro for a biosuit.

Also it looks cool in my opinion..

## Changelog
:cl:
add: Shady individuals, rejoice! A new hardsuit has been seen available for purchase on the black market, for a hefty fee. (Sprites provided by the talented Kryson)
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -426,3 +427,22 @@ Contains:
...
+
+/obj/item/clothing/suit/space/hardsuit/combatmedic
+	name = "endemic combat medic hardsuit"
+	desc = "The standard issue hardsuit of infectious disease officers, before the formation of ERT teams. This model is labeled 'Veradux'."
+	icon_state = "combatmedic"
+	item_state = "combatmedic"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/combatmedic
+	armor = list("melee" = 40, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 20, "bio" = 70, "rad" = 50, "fire" = 30, "acid" = 50)
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | SNUG_FIT
+	allowed = list(/obj/item/gun, /obj/item/melee/baton, /obj/item/twohanded/fireaxe, /obj/item/tank/internals)
```
Comment (Tlaltecuhtli) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: cant hold medicine?

Comment (ArcaneMusic) [reply code/modules/clothing/spacesuits/miscellaneous.dm]: If by "medicine" you mean a cure and by "cure" I mean a fireaxe to the head then yes.
(Easy change, 2 seconds.)

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -426,3 +427,23 @@ Contains:
...
+	hardsuit_type = "combatmedic"
+
+/obj/item/clothing/suit/space/hardsuit/combatmedic
+	name = "endemic combat medic hardsuit"
+	desc = "The standard issue hardsuit of infectious disease officers, before the formation of ERT teams. This model is labeled 'Veradux'."
+	icon_state = "combatmedic"
+	item_state = "combatmedic"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/combatmedic
+	armor = list("melee" = 40, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 20, "bio" = 70, "rad" = 50, "fire" = 30, "acid" = 50)
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | SNUG_FIT
```
Comment (Fikou) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: why did you put snug fit there

**Comment by wesoda25** [comment]
Sprite looks... tribal? Why not something a bit more futuristic looking?

**Comment by Shadowflame909** [comment]
> 
> 
> Sprite looks... tribal? Why not something a bit more futuristic looking?

Got White and Red armor colors. Looks futuristic enough to me!

**Comment by ATH1909** [comment]
Uh, I don't think bio protection does anything against zombie claw attacks/infections?

**Comment by Krysonism** [comment]
Don't all hardsuits provide perfect virus protection anyway?

**Comment by Anonmare** [comment]
Ancient reference

**Comment by anconfuzedrock** [comment]
Literally every other space suit in the entire game has 100 bio or I am a ceiling

**Comment by carlarctg** [comment]
conflicts

why doesn't it have 100 bio lol, it's spaceproof but not germsproof???

**Comment by ArcaneMusic** [comment]
Well, good news and bad news, Yes, you are correct, all other hardsuits have 100 bio protection as it stands. HOWEVER, it turns out that 100 Bio isn't ACTUALLY required to be fully disease immune!
If a disease is airborne is the only situation where a player's bio armor is ever compared, and the threshold to avoid catching the disease is LITERALLY 25 bio from what I've seen now.

Out of scope of this PR to fix this but whatever, I'm adjusting the armor values and going to find a more suitable item to slot into the hardsuit.

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 2 above)
Comment (ArcaneMusic) [reply code/modules/clothing/spacesuits/miscellaneous.dm]: Compromise between the clothing flags for the security hardsuit and the medical hardsuit. You get a LOT more information from the medical hardsuit's SCAN REAGENTS so I decided to go for this.

**Comment by cacogen** [comment]
yeah i want a free hardsuit as a non-antag but the sprite is bad can you just use the original and change the colour or something

**Comment by ArcaneMusic** [comment]
> yeah i want a free hardsuit as a non-antag but the sprite is bad can you just use the original and change the colour or something

6000 credits is hardly free, and I have some changes to the sprite done by Kryson.
Also, saying it's just "bad" isn't very constructive.

**Comment by Krysonism** [comment]
arcanehardsuit_comp
I have made a third version of the hardsuit sprite. I only have the front state so far.

**Comment by cacogen** [comment]
Looks good. Didn't realise it was so expensive

**Comment by cacogen** [comment]
btw when I criticise a PR I don't do it to make people feel bad I do it
because I want stuff to be good. And I can't sprite so I couldn't explain
the problems with the original

**Comment by cacogen** [comment]
as i said this looks good now and i didn't realise it was so expensive

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -434,3 +435,23 @@ Contains:
...
+
+//We can either be alive monsters or dead monsters. You choose.
+/obj/item/clothing/head/helmet/space/hardsuit/combatmedic
+	name = "endemic combat medic helmet"
+	desc = "The integrated helmet of the combat medic hardsuit, this has a bright, glowing facemask."
+	icon_state = "hardsuit0-combatmedic"
+	item_state = "hardsuit0-combatmedic"
+	armor = list("melee" = 35, "bullet" = 10, "laser" = 20, "energy" = 30, "bomb" = 5, "bio" = 100, "rad" = 50, "fire" = 65, "acid" = 75)
+	hardsuit_type = "combatmedic"
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | SNUG_FIT
```
Comment (Fikou) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: isnt this all inherited

**Comment by cacogen** [comment]
this pr is so old

**Pull request merged**